app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt

```diff
@@ -167,15 +167,10 @@ class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMarker
 
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_map, container, false)
+    }
 
-        var v: View? = view
-
-        try {
-            v = inflater.inflate(R.layout.fragment_map, container, false)
-        } catch (e: InflateException) {
-            Timber.d(e.localizedMessage);
-        }
-
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```

Comment:
So the fix was to use a different lifecycle method for the rest of this?

Reply:
Seems like that's what was needed.